src/main/kotlin/org/jetbrains/research/testspark/actions/llm/LLMSampleSelectorFactory.kt

```diff
@@ -172,4 +180,55 @@ class LLMSampleSelectorFactory(private val project: Project) : PanelFactory {
             testSamplePanelFactory.enabledComponents(isEnabled)
         }
     }
+
+    /**
+     * Retrieves a list of test samples from the given project.
+     *
+     * @return A list of strings, representing the names of the test samples.
+     */
```

Comment:
The implementation of this method currently supports only Java (e.g., Java-specific keywords used). Do we want to extend it for Kotlin right at the beginning, or is it for future refactoring?

Reply:
This implementation focuses exclusively on the Java language. All future refactorings and extensions to other languages will be in separate PRs